plasmapy/formulary/ionization.py

```diff
@@ -0,0 +1,55 @@
+""" This module gathers functions relating to ionization states and the properties thereof.
+"""
+__all__ = ["Z_bal"]
+
+from astropy import units as u
+from plasmapy.utils.decorators import (
+    angular_freq_to_hz,
+    check_relativistic,
+    validate_quantities,
+)
+from astropy.constants import c, a0, k_B
+from numpy import pi, exp, sqrt, log
+
+
+def Z_bal(n: u.m ** -3, T_e: u.K):
+    r"""
+    Z_bal is derived from the Saha equation with the assumptions that
+    the atoms are of a single species,
+    are either hydrogenic or completely ionized,
+    and that there is a balance between ionization and recombination,
+    meaning that the number of atoms in either state are equal.
+    The Saha equation and therefore Z_bal are more accurate when the plasma
+    is at a high density and temperature.
+
+    .. math::
+
+        Z\_bal = \sqrt{\frac{k_B T_e}{E_H}} \sqrt{\ln{\frac{1}{4 n a_{0}^3}
+        (\frac{k_B T_e}{\pi E_H})^{3/2}}} - \frac{1}{2}
+
+    Where :math:`k_B` is the Boltzmann constant,
+    :math:`a_0` is the Bohr radius, and
+    :math:`E_H` is the ionization energy of Hydrogen
+
+    Parameters
+    ----------
+
+    T_e : `~astropy.units.Quantity`
+        The electron temperature.
+
+    n : ~astropy.units.Quantity`
+        The electron number density of the plasma.
+    Returns
+    -------
+    Z: `~astropy.units.Quantity`
+        The average ionization state of the ions in the plasma
+        assuming that the number of ions in each state are equal.
+
+    """
+    E_H = 13.6 * u.eV
+
```

Comment:
Not to complain, but now I'm curious whether this would not be in astropy constants or in units as 1 Rydberg?

Reply:
I just looked and it is, I can change that